docs/user/testing-metrics.md

```diff
@@ -98,11 +98,64 @@ Note that each of these functions is marked as `internal`, you need to import `G
 
 </div>
 
+<div data-lang="Python" class="tab">
+
+It is generally a good practice to "reset" Glean prior to every unit test that uses Glean, to prevent side effects of one unit test impacting others.
+Glean contains a helper function `glean.testing.reset_glean()` for this purpose.
+It has two required arguments: the application id, and the application version.
+Each reset of Glean will create a new temporary directory for Glean to store its data in.
+This temporary directory is automatically cleaned up the next time Glean is reset or when the testing framework finishes.
+
+The instructions below assume using `py.test` as the test runner.
+Other test-running libraries have similar features, but are different in the details.
+
+To have this function run before every unit test in a `py.test` module:
+
+```python
+from glean import testing
+
+def setup_module():
+    testing.reset_glean("my-app-id", "0.1.0")
+```
+
+To have this function run before every test in an entire test suite, add the following to the `conftest.py` file:
+
+```python
+from glean import testing
+
+def pytest_runtest_module():
```

Comment:
I'm not familiar with this API and can't find anything in the pytest source or documentation.

Reply:
Sorry -- this is a typo on my end.  It's `pytest_runtest_setup`.  And I'll add a link to the pytest docs here.